### PR TITLE
Added Camera Functionality

### PR DESCRIPTION
**Pull request by craftamap**

### CODE REVIEW

File: app/src/main/java/de/hsaugsburg/teamulster/sohappy/CameraActivity.kt [hunk 1]
```diff
@@ -1,11 +1,104 @@
...
+import java.util.concurrent.Executors
 
 class CameraActivity : AppCompatActivity() {
+    companion object {
+        const val REQUEST_CODE_PERMISSIONS = 10
+    }
+
+    private var cameraProvider: ProcessCameraProvider? = null
+    private val executor = Executors.newSingleThreadExecutor()
+    private var bitmap: Bitmap? = null
```
Comment (mac641) [inline app/src/main/java/de/hsaugsburg/teamulster/sohappy/CameraActivity.kt:27]: add unified val/var order

File: app/src/main/java/de/hsaugsburg/teamulster/sohappy/CameraActivity.kt [hunk 2]
```diff
@@ -1,11 +1,104 @@
...
+import androidx.camera.core.ImageAnalysis
+import androidx.camera.lifecycle.ProcessCameraProvider
+import androidx.core.content.ContextCompat
+import de.hsaugsburg.teamulster.sohappy.util.YuvToRgbConverter
+import jp.co.cyberagent.android.gpuimage.GPUImage
+import jp.co.cyberagent.android.gpuimage.GPUImageView
+import java.util.concurrent.Executors
 
 class CameraActivity : AppCompatActivity() {
+    companion object {
```
Comment (mac641) [inline app/src/main/java/de/hsaugsburg/teamulster/sohappy/CameraActivity.kt:21]: add comments in whole code for better understanding

**Review by mac641 — CHANGES_REQUESTED** [review]

**Review by limkokyew — APPROVED** [review]
Looks good to me.

**Review by AnnaGsell — APPROVED** [review]

**Review by diehmlukas — APPROVED** [review]

**Review by mac641 — APPROVED** [review]
Comments looking good 👍🏿

**Pull request merged**